src/universal/components/RetroReflectPhase/PhaseItemColumn.js

```diff
@@ -14,38 +14,38 @@ import AddReflectionButton from 'universal/components/AddReflectionButton/AddRef
 import ui from 'universal/styles/ui';
 import {GROUP, REFLECT, REFLECTION_CARD, VOTE} from 'universal/utils/constants';
 import ReflectionGroup from 'universal/components/ReflectionGroup/ReflectionGroup';
-import type {DroppableProvided, DroppableStateSnapshot} from 'react-beautiful-dnd/src/index';
-
-import {Droppable} from 'react-beautiful-dnd';
 import {createFragmentContainer} from 'react-relay';
 import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
 import reactLifecyclesCompat from 'react-lifecycles-compat';
-import ReflectionDropZone from 'universal/components/ReflectionDropZone';
 import ReflectionCard from 'universal/components/ReflectionCard/ReflectionCard';
 import AnonymousReflectionCard from 'universal/components/AnonymousReflectionCard/AnonymousReflectionCard';
 
 import {LabelHeading} from 'universal/components';
+import {DropTarget as dropTarget} from 'react-dnd';
+import withMutationProps from 'universal/utils/relay/withMutationProps';
+import dndNoise from 'universal/utils/dndNoise';
+import UpdateReflectionLocationMutation from 'universal/mutations/UpdateReflectionLocationMutation';
+import appTheme from 'universal/styles/theme/appTheme';
 
 const ColumnWrapper = styled('div')({
   alignItems: 'center',
   display: 'flex',
   flexDirection: 'column',
-  flex: 1
+  flex: 1,
+  height: '100%'
 });
 
 const ReflectionsArea = styled('div')({
   flexDirection: 'column',
   display: 'flex',
-  // cannot use overflow since react-beautiful-dnd does not use portals.
-  // overflow: 'auto',
+  overflow: 'auto',
   height: '100%',
   minWidth: ui.retroCardWidth
 });
 
-const ReflectionsList = styled('div')({
-  // adds a buffer to the dropzone to limit unwanted drags to the dropzone
-  marginBottom: '1rem'
-});
+const ReflectionsList = styled('div')(({canDrop}) => ({
+  background: canDrop && appTheme.palette.light60l
```

Comment:
subtle column highlighting to show that you can ungroup if you drop the reflection. @ackernaut you may wanna change this